src/backend/controllers/kubernetes/ingress/ingress.go

```diff
@@ -0,0 +1,128 @@
+package ingress
+
+import (
+	"encoding/json"
+
+	"github.com/Qihoo360/wayne/src/backend/client"
+	"github.com/Qihoo360/wayne/src/backend/controllers/base"
+	"github.com/Qihoo360/wayne/src/backend/models"
+	"github.com/Qihoo360/wayne/src/backend/resources/ingress"
+	"github.com/Qihoo360/wayne/src/backend/util/logs"
+	kapiv1beta1 "k8s.io/api/extensions/v1beta1"
+)
+
+type KubeIngressController struct {
+	base.APIController
+}
+
+func (c *KubeIngressController) URLMapping() {
+	c.Mapping("Get", c.Get)
+	c.Mapping("Offline", c.Offline)
+	c.Mapping("Deploy", c.Deploy)
+}
+
+func (c *KubeIngressController) Prepare() {
+	c.APIController.Prepare()
+	perAction := ""
+	_, method := c.GetControllerAndAction()
+	switch method {
+	case "Get":
+		perAction = models.PermissionRead
+	case "Deploy":
+		perAction = models.PermissionDeploy
+	case "offline":
+		perAction = models.PermissionOffline
+	}
+	if perAction != "" {
+		c.CheckPermission(models.PermissionTypeService, perAction)
+	}
+}
+
```

Comment:
Please add a comment for the business code function.(Just as you do in src/backend/controllers/ingress/ingress_tpl.go)